### ISSUE TITLE
Feature Request: Unknown Types

### ISSUE THREAD

**Issue by quardbreak** [SpaiR/StrongDMM#170]
### Problem to be solved

This issue for a reminder and progress tracking. Current StrongDMM alpha version lack of unknown types resolver feature, which allows to open incompatible maps and replace invalid paths/variables for easier conversion.

### Sugest a solution

Add resolver screen back (close to same as before). Some optional suggestions:
* Lock cell height for each error. On pre-alpha version there was a visual problem if window has a lot errors. This look inconvenient, sometimes you have to pull down lines to see all variables because they're squeezed from window open. If someone don't get what I mean, [check this screenshot](https://user-images.githubusercontent.com/60765218/105199067-547ac800-5b0c-11eb-92e7-6b20cabce388.png).
* Should be accessible without scrolling whole list.
* * Add counters of total found errors and current resolve progress. Maybe in window's header/title, if they allow runtime update?
* * Fold/unfold all errors.
* https://github.com/SpaiR/StrongDMM/issues/85 have a good suggestion.
* * Actually I'd like to add my suggestion regarding this: skip button to skip everything, and skip button to skip specific error.

**Comment by quardbreak** [comment]
Any progress on this?

**Comment by SpaiR** [comment]
Nope. And won't happen until someone will do this. My position about SDMM maintaining is here https://github.com/SpaiR/StrongDMM/issues/161#issuecomment-1719047455 It's applicable to features requests even more.
